### DISCUSSION TITLE
Is there a place to Share or Download decks that others have made?

### DISCUSSION THREAD

**Discussion by Jeager2** [st3v3nmw/obsidian-spaced-repetition#553] [Q&A]
Is there a place to Share or Download decks that others have made?

**Comment by st3v3nmw** [comment]
Currently, no. We could explore ways to support that

**Comment by Jeager2** [comment]
Wow thanks for responding so quickly. 
I have a few suggestions to keep data organized as people enter it.
1) Use the Dewey Decimal system to separate decks into their respective Areas and sub Areas
2) Facilitate the ability for registered users to Thumb up or down a Deck or specific card
3) Allow users to comment on decks so that others may read them and carry on discussions
4) Allow users to comment on Cards so that others may read them and carry on discussions
5) Make sure to be able to remove a bad actors actions on decks, cards, comments etc.

**Comment by Jeager2** [comment]
Not only will you have spaced repetition learning, but now people can share and reduce duplication of work.

**Comment by Jeager2** [comment]
for reference: https://en.wikipedia.org/wiki/List_of_Dewey_Decimal_classes
